test/ci/verify-templates.test.js

```diff
@@ -0,0 +1,137 @@
+'use strict';
+
+// These tests run in a CI context and verify that some important guarantees
+// hold for all the functions in the repository.
+
+if(process.env.CI) {
+    const path = require('path');
+    const fs = require('fs');
+    const { parser } = require('configure-env');
+
+    const excludedPaths = ['node_modules', 'test', 'coverage', 'docs'];
```

Comment:
I like including `blank` in this list, since it's intentionally non-QD. For the rest, I want to define a `skipList` or similar array that will contain the names of templates that aren't QD yet, but will be down the line. Eventually `skipList` will be empty, and we'll remove it from the test suite.